data_collection/web/guides.rst

```diff
@@ -221,187 +221,184 @@ indicate no value.
-    _paq.push(["addEcommerceItem", "82775027", "MEGA Milk Chocolate 200g", "Candy", 6.00, 1]);
-    _paq.push(["trackEcommerceCartUpdate", 6.00]);
-
-    // visitor added one mango fruit to a shopping cart with one chocolate bar
-    _paq.push(["addEcommerceItem", "01809926", "FRUTASTIC Mango", "Fruits & vegetables", 4.00, 1]);
-    _paq.push(["trackEcommerceCartUpdate", 10.00]);
-
-    // visitor removed one chocolate bar from a shopping cart with one chocolate bar and one mango
-    _paq.push(["removeEcommerceItem", "82775027"]);
-    _paq.push(["trackEcommerceCartUpdate", 4.00]);
+  _paq.push([
+      "ecommerceCartUpdate",
+      [
+          {
+              sku: "craft-311",
+              name: "Unicorn Iron on Patch",
+              category: "Crafts & Sewing",
+              price: "50.00",
+              quantity: 3,
+              brand: "DMZ",
+              variant: "blue"
+          }
+          {
+              sku: "craft-312",
+              name: "Unicorn Iron on Grass",
+              category: "Crafts & Sewing",
+              price: "30.00",
+              quantity: 1,
+              brand: "DMZ",
+              variant: "red"
+          }
+      ],
+      "180.00"
+  ]);
+
+
+Product addition and removal commands are used immidiately when visitor adds to a cart or removes from a cart any product.
+These commands let us track how visitor interacted with a cart and update a cart state.
+
+Adding prodict to a cart:
+
+.. code-block:: javascript
+
+  _paq.push([
+      "ecommerceAddToCart",
+      {
+          sku: "craft-311",
+          name: "Unicorn Iron on Patch",
+          category: "Crafts & Sewing",
+          price: "49.90",
+          quantity: 3,
+          brand: "DMZ",
+          variant: "blue"
+      }
+  ]);
+
+Removing product from a cart:
+
+.. code-block:: javascript
+
+  _paq.push([
+      "ecommerceRemoveFromCart",
+      {
+          sku: "craft-311",
+          name: "Unicorn Iron on Patch",
+          category: "Crafts & Sewing",
+          price: "49.90",
+          quantity: 3,
+          brand: "DMZ",
+          variant: "blue"
+      }
+  ]);
 
 .. _guide_tracking_orders:
 
 Tracking orders
 ^^^^^^^^^^^^^^^
 
-Perhaps the most important element of an e-commerce implementation is tracking orders. Just like with :ref:`cart updates<guide_tracking_cart_updates>`, tracking orders has two steps: registering items that have been purchased and tracking the order. Registering items looks exactly the same - we use :ref:`addEcommerceItem<jtc-api-addEcommerceItem>`, :ref:`removeEcommerceItem<jtc-api-addEcommerceItem>` and :ref:`clearEcommerceCart<jtc-api-clearEcommerceCart>`. The actual tracking of an order is done with a call to :ref:`trackEcommerceOrder<jtc-api-trackEcommerceOrder>` function.
-
-.. code-block:: javascript
+Perhaps the most important element of an e-commerce implementation is tracking orders.
+E-commerce v2 API is using :ref:`ecommerceConversion<jtc-api-ecommerceConversion>` for this task.
```

Comment:
```suggestion
E-commerce API offers :ref:`ecommerceConversion<jtc-api-ecommerceConversion>` for this task.
```
In the first comment @harmasz suggested no to call it `v2` - thus my code suggestion here.